src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBscOptions.vue

```diff
@@ -2,7 +2,7 @@
   <div class="flex justify-between text-sm">
     <div><span class="text-gray-400">Swap > BSC > </span> <span class="text-blue-primary font-bold">Select option</span></div>
     <div>
-      <router-link :to="{ name: 'ViewServices'}" class="font-bold" active-class="accounts">All Services</router-link>
+      <router-link :to="{ name: 'ViewServices'}" class="font-bold" active-class="accounts">Home</router-link>
```

Comment:
this Home can be apply to the ts-base ?

Reply:
@laics1984 please confirm, thanks

Reply:
@sleepyOwl14 Title has been updated. Thanks